apps/hygeia_web/lib/hygeia_web/controllers/locale_controller.ex

```diff
@@ -12,6 +12,19 @@ defmodule HygeiaWeb.LocaleController do
 
     conn
     |> put_session(SetLocale.session_key(), locale)
+    |> maybe_set_language_warning(locale)
     |> redirect(external: redirect_uri)
   end
+
+  defp maybe_set_language_warning(conn, locale) do
+    # TODO: Extract to configuration
```

Comment:
`config/config.exs`

```elixir
config :hygeia_gettext, HygeiaGettext, fuzzy_languages: ["it", "fr"]
```

`apps/hygeia_gettext/lib/hygeia_gettext.ex`

```elixir
@fuzzy_languages Application.compile_env(:hygeia_gettext, [__MODULE__, :fuzzy_languages], [])

@spec if_fuzzy_language?(language :: String.t()) :: boolean
def if_fuzzy_language?(language), do: language in @fuzzy_languages
```

```suggestion
    if  HygeiaGettext.is_fuzzy_language?(locale) do
```